src/main/lrsql/init/oidc.clj

```diff
@@ -128,11 +130,15 @@
   "Given a webserver config, return a (possibly empty) vector of interceptors
   for use with the admin API. These validate token claims and ensure an admin
   account is made."
-  [{:keys [oidc-issuer]}]
+  [{:keys [oidc-issuer
+           oidc-enable-local-admin]}]
```

Comment:
I notice that you use `oidc-enable-local-admin` in multiple places in the code, in multiple namespaces. I feel like this is fairly brittle, since you would have multiple places where you could introduce bugs relating to this config variable. I'm personally not sure what's the best way to refactor this, but it's something to consider.

Reply:
I'm not sure what the problem is? It is a config that is needed in several places, so I use it.

Reply:
Sure, but if you could reduce the number of places it's needed in, I think it would be great (even if it's impossible to only use it in a single place).

Reply:
Sometimes complex features are just complex, yeah.